src/highscore.cpp

```diff
@@ -98,7 +98,7 @@ int8 SaveHighScoreValueNetwork()
 	/* Sort all active companies with the highest score first */
 	FOR_ALL_COMPANIES(c) cl[count++] = c;
 
-	QSortT(cl, count, &HighScoreSorter);
+	std::sort(std::begin(cl), std::begin(cl) + count, HighScoreSorter);
```

Comment:
`std::end()` instead of `std::begin(cl) + count´?

Reply:
No because only `count` elements are valid, due to how `FOR_ALL_COMPANIES()` works